Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -40,23 +40,28 @@ public struct Configuration: Equatable {
                  excluded: [String] = [],
                  warningThreshold: Int? = nil,
                  reporter: String = XcodeReporter.identifier,
-                 configuredRules: [Rule] = masterRuleList.configuredRules(with: [:])) {
+                 ruleList: RuleList = masterRuleList,
+                 configuredRules: [Rule]? = nil) {
+
         self.included = included
         self.excluded = excluded
         self.reporter = reporter
 
+        let configuredRules = configuredRules ?? ((try? ruleList.configuredRules(with: [:])) ?? [])
```

Comment:
The superfluous parens were my attempt of making it more readable. Much better on multiple lines!